src/main/java/org/czekalski/petapiintegration/apiClient/v1/dto/PaginationDto.java

```diff
@@ -0,0 +1,46 @@
+
+package org.czekalski.petapiintegration.apiClient.v1.dto;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.fasterxml.jackson.annotation.*;
+
+public class PaginationDto {
+    private final Integer countPerPage;
+
+    private final Integer totalCount;
+
+    private final Integer currentPage;
+
+    private final Integer totalPages;
```

Comment:
Ta sama historia, co z `Boolean`, jeżeli nie zamierzasz umieszczać w tych polach `null`, to powinny być `int`

Reply:
ok, poprawię